material/src/test/java/org/odk/collect/material/ErrorsPillTest.kt

```diff
@@ -0,0 +1,54 @@
+package org.odk.collect.material
+
+import android.app.Application
+import android.view.View
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.CoreMatchers.equalTo
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.Robolectric
+import org.robolectric.Shadows
+
+@RunWith(AndroidJUnit4::class)
+class ErrorsPillTest {
+    private val context = ApplicationProvider.getApplicationContext<Application>().also {
+        it.setTheme(com.google.android.material.R.style.Theme_Material3_Light)
+    }
+    private val attrs = Robolectric.buildAttributeSet().build()
```

Comment:
I think you should be able to just pass `null` for `attrs`.

Reply:
Right.